_cards/features.md

```diff
@@ -11,7 +11,9 @@ urls:
   getstarted: "#Onboarding"  
 ---
 <div class="row">
-  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">Using common services will jump start the development efforts of your team and lower the overall maintenance cost of your product. You can focus on building features that matter to your users. Get started today!</div>
+  <div class="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-xs-12">
```

Comment:
Can be replaced with "col-lg-6 col-xs-12" as only the lower-bound breakpoint is required

Reply:
done